### DISCUSSION TITLE
Remote Control

### DISCUSSION THREAD

**Discussion by NekoTheWeird** [pear-devs/pear-desktop#335] [Ideas]
If possible add a remote control (Something like spotify has). Tho im not sure from what to control it since i bet you will not make an app just for this. Would be real nice tho :3

**Comment by Bellisario** [comment]
@NekoTheWeird, the problem is that it's very complicate to create a "remote control", but I think that could be implemented a plugin that makes the same actions in more than one computer where one reproduces the songs and the second receive data from the first (ex. info about playing, next songs etc.) and doesn't reproduces anything, but it sends your action to the first one saying what to do!
I repeat what I say, it's very very much complex!!!

**Comment by OtakuDragon53** [comment]
the original youtube music desktop app has one. I'm trying to see if this project is compatible with it as I need it for integration with Touch Portal. is there a way that you could add it?

**Reply by DeerAndy** [reply]
I'm looking for this but so I can connect it to Streamer.bot so it can output what I'm listening to into my chat.

**Comment by VBtheFirbolg** [comment]
This is an aside and might not pertain directly to the discussion at hand, but you could use the MPRIS interface in Linux distributions and something like KDE Connect which does serve as a very handy remote (albeit with limited features).

**Comment by YellowWolf** [comment]
Touch portal

**Comment by devangerpl** [comment]
Will you make support for touch portal apps? I noticed that some other apps already work with touch portal.
